Files/Filesystem/Drives.cs

```diff
@@ -288,6 +288,29 @@ private void GetVirtualDrivesList(IList<DriveItem> list)
             }
         }
 
+        public static async Task<StorageFolderWithPath> GetRootFromPath(string devicePath)
```

Comment:
@tsvietOK could you put a breakpoint on this method (GetRootFromPath) and see why it returns `null`? (when you navigate to the phone storage)

Reply:
devicePath = "\\\\?\\ONEPLUS A5010\\"
![image](https://user-images.githubusercontent.com/20501502/85863881-70afc400-b7cc-11ea-9e49-73c13c80af82.png)
matchingDrive == null, so this method returns null

Reply:
Thanks a lot. It's not finding a device with matching name so it returns null. Previous method for getting the drive was more robust, guess I'll revert to that.

Reply:
@tsvietOK issue should now be fixed, please let me know.